### DISCUSSION TITLE
How to use unocss for developing a component library?

### DISCUSSION THREAD

**Discussion by qixiaobro** [unocss/unocss#2784] [Q&A]
Hello, I want to use unocss to develop a component library instead of using the traditional way of developing with CSS files. However, I have encountered an issue where if I don't include the styles in the safelist, the corresponding component styles won't be generated during packaging. How can I solve this problem? Do I have to put them in the safelist? Thank you!

**Comment by siduck** [comment]
hi @qixiaobro did you fix this?
